pages/learn/develop/hardware/i2c-and-spi.md

```diff
@@ -348,6 +350,16 @@ After the custom device tree has been validated, it can be included in newer bal
 
 Please note that if the changes for your carrier board expand past kernel device-trees, or require modifications to board configuration files like pin multiplexing configuration files or any other device-trees or files used by firmware, these may not be provided by the existing cloud images. During provisioning the resulting configuration changes are stored in the QSPI or in the hardware defined boot partitions, and thus will be replaced with the default values when updating the Host Operating System. Please [contact us](https://www.balena.io/contact-sales) if you would like to use a Jetson carrier board which may not be fully compatible with its' corresponding devkit, or with any of our cloud images for your Jetson module.
 
+### Configurable fan profiles
+
+Jetson Orin Devices running balenaOS revisions newer than <TODO> and supervisor versions greater than <TODO> offer support for configurable fan profiles. You can switch between the two options provided by Jetpack: "quiet" and "cool", by navigating to the "Device Configuration" page on the balenaCloud dashboard and selecting the desired option. The change is performed at runtime, and does not trigger a device reboot. Preloading the [fan profile configuration](/reference/OS/configuration/#fanprofile) before provisioning your device can also be achieved by editing [config.json](/reference/OS/configuration/#about-configjson)
```

Comment:
```suggestion
Jetson Orin Devices running balenaOS revisions newer than <TODO> and supervisor versions greater than <TODO> offer support for configurable fan profiles. You can switch between the options provided by Jetpack by navigating to the Device/Fleet Configuration tab on the sidebar in the balenaCloud dashboard and selecting the desired option. The change will be performed during runtime and doesn't trigger a device reboot. Preloading the [fan profile configuration](/reference/OS/configuration/#fanprofile) before provisioning your device can be achieved by editing the [config.json](/reference/OS/configuration/#about-configjson) file.
```

1. Let's not mention options mentioned in the docs on the account of Nvidia making a 3rd mode called "High power" and then that prompts us to correct our docs. Looking into the future this decreases our maintenance. 
2. Additionally, I am right to assume this is also a fleet configuration option?
3. Need to be specific at places.

Reply:
@vipulgupta2048  1) & 3) are addressed. @cywang117 can you please confirm if these configurations will be available for fleets as well?

Reply:
These configuration options are available both at fleet and device level @vipulgupta2048